kratos/python_scripts/line_output_process.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
how is this fitting with the rest of the PR?

Reply:
Will move it to a separate PR